setup.py

```diff
@@ -29,6 +30,19 @@ def read_requirements(path):
     return [line.strip() for line in read(path).split("\n") if not line.startswith(('"', "#", "-"))]
 
 
+def get_version():
+    """Looks for __version__ attribute in top most __init__.py"""
+    version_lines = [l for l in read("examples_utils/__init__.py").splitlines() if re.match("__version__\\s*=", l)]
```

Comment:
Would it not suffice to just look for `"__version__" in l for l in read(file).splitlines()` rather than importing and using regex? I guess regex is more robust and possibly even reccommended but it feels like overkill here no?

Reply:
I didn't want the regexp but I felt that I wanted reliability if someone changed the white space. I also wanted to be resilient to a comment string including `__version__`.